rm-datasource/src/main/java/io/seata/rm/datasource/exec/UpdateExecutor.java

```diff
@@ -137,18 +134,19 @@ protected List<String> getNeedUpdateColumns(String table, String tableAlias, Lis
                     needUpdateColumns.add(getColumnNamesWithTablePrefix(table,tableAlias,pkNameList));
                 }
             }
-            needUpdateColumns.addAll(originUpdateColumns);
+            needUpdateColumns.addAll(originUpdateColumns.parallelStream()
```

Comment:
We want the variable names to be more formal, such as the originUpdateColumns variable followed by Unescape.

Reply:
done